RecFCCeeCalorimeter/src/components/AugmentClustersFCCee.cpp

```diff
@@ -77,22 +87,55 @@ StatusCode AugmentClustersFCCee::initialize()
       showerShapeDecorations.push_back(Form("energy_fraction_%s_layer_%d", detector, layer));
       showerShapeDecorations.push_back(Form("theta_%s_layer_%d", detector, layer));
       showerShapeDecorations.push_back(Form("phi_%s_layer_%d", detector, layer));
+      showerShapeDecorations.push_back(Form("width_theta_%s_layer_%d", detector, layer));
```

Comment:
the idea was that these decorations would be added conditionally on a boolean flag. can you add it?